docs/guides/02-concepts/security.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
you probably have an idea already, but with the new-ish security page we need a good way to (re-)integrate the second half of the old one (starting here <https://deploy-preview-590--tauri.netlify.app/v1/guides/development/security#tauri-api>) _somewhere_ (it would be good if they'd be easily accessible in/from the corresponding js module docs).

Reply:
There's a yet-to-be migrated sub-page for security:
https://github.com/JonasKruckenberg/tauri-docs-wip/blob/main/src/background/security/context-isolation.md
Should we break out https://github.com/tauri-apps/tauri-docs/blob/dev/docs/guides/development/security.md#tauri-api into a separate page and add it under `concepts/security`?

If there are any things we want to highlight outside of concepts (imagining if someone skips concepts section), then I think we should add that directly in the API documentation

Reply:
I meant like the tauri rust/js (in-code) API docs. Keep everything in background/security like it is now, but if there are any very important bits, those should ALSO be in our API docs where relevant.